cli/cli.ts

```diff
@@ -717,7 +717,6 @@ function targetFileList() {
     if (simDir() != "sim")
         lst = lst.concat(nodeutil.allFiles(path.join("sim", "public"), 5, true))
     pxt.debug(`target files (on disk): ${lst.join('\r\n    ')}`)
-    lst = lst.filter(f => !f.includes("icons.css")) // We want to pull icons.css from pxt-core, not the target
```

Comment:
Cloud icon still there?

Reply:
yup with https://github.com/microsoft/pxt-arcade-sim/pull/50

Reply:
Oh! Do we need to do anything special to bump pxt-arcade-sim?

Reply:
hmmmmmmm i think @riknoll knows? not sure what being an unpublished npm package means...

Reply:
You need to manually do a bump (change the version in package.json and push a tag).

Then you update the version in pxt-arcade in three places: The package.json and in tow of the github actions builds. Here's an example PR: https://github.com/microsoft/pxt-arcade/pull/3034/files